src/amo/components/SectionLinks/index.js

```diff
@@ -66,30 +66,11 @@ export class SectionLinksBase extends React.Component<InternalProps> {
   render(): React.Node {
     const { className, clientApp, forBlog, i18n, viewContext } = this.props;
 
-    let forBrowserNameText;
-    if (clientApp === CLIENT_APP_FIREFOX) {
-      forBrowserNameText = i18n.gettext('for Firefox');
-    } else if (clientApp === CLIENT_APP_ANDROID) {
-      forBrowserNameText = i18n.gettext('for Android');
-    }
-
-    const sectionsForBrowser = [];
-
-    if (clientApp !== CLIENT_APP_ANDROID) {
-      sectionsForBrowser.push(
-        <DropdownMenuItem key="dictionaries-and-language-packs">
-          <Link
-            className={makeClassName('SectionLinks-dropdownlink', {
-              'SectionLinks-dropdownlink--active':
-                viewContext === VIEW_CONTEXT_LANGUAGE_TOOLS,
-            })}
-            to="/language-tools/"
-          >
-            {i18n.gettext('Dictionaries & Language Packs')}
-          </Link>
-        </DropdownMenuItem>,
-      );
-    }
```

Comment:
I understand. I normally wouldn't have done it, but we ended up with a code coverage gap because moving from shallow rendering to testing this where it's actually used revealed that we have dead code. I guess I could have added comments and made it a follow-up. I will try to do that in the future. One issue with follow-ups is that sometimes they just never get addressed.